app/src/main/java/com/ddoczi/tasky/authentication/presentation/composables/BaseIcon.kt

```diff
@@ -0,0 +1,70 @@
+package com.ddoczi.tasky.authentication.presentation.composables
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import com.ddoczi.tasky.R
+import com.ddoczi.tasky.authentication.enums.InputFieldType
+import com.ddoczi.tasky.ui.theme.Gray
+import com.ddoczi.tasky.ui.theme.Green
+import com.ddoczi.tasky.ui.theme.Red
+
+@Composable
+fun BaseIcon(
```

Comment:
Not a big fan of the name `BaseIcon` (generally not a big fan of calling things `BaseX`), since that doesn't communicate what someone could expect from this composable. In this specific case this icon seems to be only for text fields, but the name base would let me think it's for any icon in the app.
What about `TaskyTextFieldIcon` instead? Makes it clear that it's an icon adjusted to Tasky's style and is used for text fields only